src/Resources/translations/CmfSonataAdminIntegrationBundle.it.xliff

```diff
@@ -286,14 +284,12 @@
             <!-- $Core -->
             <trans-unit id="form.tab_publish">
                 <source>form.tab_publish</source>
-                <!-- TODO: Translate -->
-                <target>Publish Settings</target>
+                <target>Pubblicazione</target>
             </trans-unit>
 
             <trans-unit id="form.group_publish_workflow">
                 <source>form.group_publish_workflow</source>
-                <!-- TODO: Translate -->
-                <target>Status</target>
+                <target>Workflow pubblicazione</target>
```

Comment:
Ok, then I'm going to translate the word "status", instead of "publication workflow".